From ec1bdb8e0beb5c8a539c90ac32aa40c8c8190a41 Mon Sep 17 00:00:00 2001
From: tropicrainforest <tropicrainforest27017@gmail.com>
Date: Sun, 7 Jun 2026 16:54:34 +0200
Subject: [PATCH 24/28] pcre: drop ERROR_CONDITION_ATOMIC_ASSERTION_EXPECTED
 (removed in pcre2 10.45)

MongoDB vendors pcre2 10.40, whose header defines the error constant
PCRE2_ERROR_CONDITION_ATOMIC_ASSERTION_EXPECTED (value 198), and its error-name
table references it. pcre2 removed that constant in 10.45 and reused code 198 for
a different error, and conda-forge now ships 10.47.
---
 src/mongo/util/pcre.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mongo/util/pcre.h b/src/mongo/util/pcre.h
index 5f565291..3b03c2f4 100644
--- a/src/mongo/util/pcre.h
+++ b/src/mongo/util/pcre.h
@@ -155,7 +155,6 @@ namespace mongo::pcre {
     X(ERROR_ALPHA_ASSERTION_UNKNOWN)             \
     X(ERROR_SCRIPT_RUN_NOT_AVAILABLE)            \
     X(ERROR_TOO_MANY_CAPTURES)                   \
-    X(ERROR_CONDITION_ATOMIC_ASSERTION_EXPECTED) \
     X(ERROR_BACKSLASH_K_IN_LOOKAROUND)           \
     X(ERROR_NOMATCH)                             \
     X(ERROR_PARTIAL)                             \
